src/test_utils/stub_browser_storage.js

```diff
@@ -1,92 +1,109 @@
-const keys = Symbol('keys');
-const values = Symbol('values');
-const remainingSize = Symbol('remainingSize');
-
 export default class StubBrowserStorage {
   constructor() {
-    this[keys] = [];
-    this[values] = [];
-    this[remainingSize] = 5000000; // 5mb, minimum browser storage size
+    this._keys = [];
+    this._values = [];
+    this._size = 0;
+    this._sizeLimit = 5000000; // 5mb, minimum browser storage size
```

Comment:
It's basically just the minimum standard size browsers use. Since this is a stub it's not really used